cylc/uiserver/resolvers.py

```diff
@@ -172,80 +166,72 @@ def func(schema_key, schema_value):
     converters = OPT_CONVERTERS[schema]
     api_opts = {}
     for opt, value in schema_opts.items():
-        # All valid options should be in SCHEMA_TO_API:
-        if opt not in converters:
-            raise InvalidSchemaOptionError(
-                f'{opt} is not a valid option for Cylc Clean'
-            )
```

Comment:
Instead of this we can rely on the Python API `Options` class from `cylc.flow.optparse` (in this instance, `CleanOptions`)